src/index.js

```diff
@@ -28,7 +28,7 @@ app.use((req, res, next) => {
 if (!isProduction) {
   app.use((err, req, res) => {
     // eslint-disable-next-line no-console
-    console.log(err.stack);
+    generateToken(err.stack);
```

Comment:
Why are you using `generateToken` here?

Reply:
That was a mistake. Did search and replace. I've fixed it